### ISSUE TITLE
cross validation mrIMLpredicts for regression

### ISSUE THREAD

**Issue by statslover123** [nickfountainjones/mrIML#6]
Hello

I am uncertain on how the k parameter works, your description here states 
k | A numeric sets the number of folds in the 10-fold cross-validation. 10 is the default.

Does that mean the default is 10 repetitions of 10-fold cross validation? (100 total test runs) or one repetition of 10 fold cross validation?

**Comment by nfj1380** [comment]
Hi there,

One repetition of 10 folds (k is just the number of folds). Sorry for the confusion - will fix this.

Cheers,

Nick

**Comment by statslover123** [comment]
that is awesome thank you, if I am tuning a nnet for regression, I can specify my hyperparameters in a grid and then use what number of folds I want to test with, using this command? thank you for the help

**Comment by nfj1380** [comment]
Yes that's right! On the topic of nnets - at least for classification problems 'brulee' MLP models now work https://brulee.tidymodels.org/reference/brulee_mlp.html i.e 
model_dnn <-
  mlp(
    hidden_units = 10, # 10 to start with
    dropout      = tune(),
    epochs       = tune(),
    learn_rate   = tune(),
    activation   = "elu"
  ) %>% 
  set_engine("brulee") |>
  set_mode("classification")

May or may not be useful for regression problems.

**Comment by statslover123** [comment]
thank you! trying this now finally
